catalog/nsili/catalog-nsili-orb-impl/src/main/java/org/codice/alliance/nsili/orb/impl/CorbaOrbImpl.java

```diff
@@ -83,21 +107,18 @@ public void shutdown() {
         orb = null;
     }
 
-
     public void refresh(Map<String, Object> configuration) {
         if (MapUtils.isEmpty(configuration)) {
             LOGGER.error("Received null or empty configuration during refresh.");
             return;
         }
 
-        Integer corbaTimeout = (Integer) configuration.get(CORBA_TIMEOUT);
-        if (corbaTimeout != null && corbaTimeout != this.corbaTimeout) {
-            setCorbaTimeout(corbaTimeout);
+        if (configuration.get(CORBA_TIMEOUT) != null) {
+            setCorbaTimeout((Integer) configuration.get(CORBA_TIMEOUT));
```

Comment:
I'm not sure if this strictly necessary because we know the types of the variables in advance from the metatype file. I'd be interested to know these refresh methods are ever called (probably very bad practice) from something else other whatever does the updating with metatype configurations.